.github/workflows/linux.yml

```diff
@@ -177,25 +177,19 @@ jobs:
         with:
            submodules: recursive
 
-      - name: install boost
-        run: |
-          sudo apt install libboost-tools-dev libboost-python-dev libboost-dev libboost-system-dev
-          echo "using gcc ;" >>~/user-config.jam
-          echo "using python ;" >>~/user-config.jam
```

Comment:
Is there a good reason not to configure boost build? It seems risky to rely on its default heuristic to pick gcc

Reply:
This was intentional; the typical expectation with setuptools is that it should be able to pick sane defaults without special configuration (except for installing dependencies to build extensions).

I could add a separate test to pass `toolset=gcc`. I'm still trying to get a feel for what other specific feature properties to test.

Reply:
I would like to preserve the test of `b2` in the `bindings/python` directory. would you mind restoring this and adding another job that runs `setup.py`?

Reply:
I would like to keep this `python` job. would you mind just reverting this file?